src/Plugin/formatter/FormatterJson.php

```diff
@@ -27,17 +39,17 @@ public function prepare(array $data) {
 
     $output = array('data' => $data);
```

Comment:
not in the scope of this PR (and sorry for doing the review post-merge), but we should allow the restful resource inject not only the 'data' property but also other properties (e.g. ``'total posts by author'``)